[BUG] multiple IT cases failed exceeded the timeout of 900 seconds

**Describe the bug**
e.g.
rapids_integration-pre_release-github, run: 834
rapids_it-arm64-pre_release, run: 186
```
org.apache.spark.SparkException: Job 37776 cancelled RAPIDS Integration Test Job 37776 exceeded the timeout of 900 seconds, cancelling. Look into fixing the test or reducing its execution time. If necessary, adjust the timeout using the marker pytest.mark.spark_job_timeout(seconds,dump_threads)
FAILED ../../src/main/python/sort_test.py::test_large_orderby[OUTOFCORE-Array(Byte)][DATAGEN_SEED=1743146821, TZ=UTC]
```

rapids-it-dataproc-2.1-ubuntu20, run: 479
```
Job 0 cancelled RAPIDS Integration Test Job 0 exceeded the timeout of 900 seconds, cancelling. Look into fixing the test or reducing its execution time. If necessary, adjust the timeout using the marker pytest.mark.spark_job_timeout(seconds,dump_threads)
FAILED ../../src/main/python/subquery_test.py::test_scalar_subquery_basics[Byte][DATAGEN_SEED=1743155641, TZ=UTC, INJECT_OOM, IGNORE_ORDER({'local': True})]
```

**Steps/Code to reproduce bug**
Please provide a list of steps or a code sample to reproduce the issue.
Avoid posting private or sensitive data.

**Expected behavior**
Pass the case. Update the case or adjust the timeout

**Environment details (please complete the following information)**
 - Environment location: [Standalone, YARN, Kubernetes, Cloud(specify cloud provider)]
 - Spark configuration settings related to the issue

**Additional context**
Add any other context about the problem here.

--- a/integration_tests/src/main/python/spark_init_internal.py
+++ b/integration_tests/src/main/python/spark_init_internal.py
@@ -261,7 +261,7 @@ def log_test_name(request):
 def set_spark_job_timeout(request):
     # TODO dial down after identifying all long tests
     # and set exceptions there
-    default_timeout_seconds = 900
+    default_timeout_seconds = 60 * 60
     logger.debug("set_spark_job_timeout: BEFORE TEST\n")
     tm = request.node.get_closest_marker("spark_job_timeout")
     if tm:

